packages/backend-proxy-middleware/src/base/proxy.ts

```diff
@@ -0,0 +1,347 @@
+import HttpsProxyAgent from 'https-proxy-agent';
```

Comment:
I believe that this needs to be `import { HttpsProxyAgent } from 'https-proxy-agent';` I had issue with this import statement some time ago.